optimum/onnxruntime/modeling_decoder.py

```diff
@@ -649,19 +650,22 @@ def forward(
                 input_ids=input_ids,
                 attention_mask=attention_mask,
                 past_key_values=past_key_values,
+                position_ids=position_ids,
```

Comment:
`self.decoder` is an `ORTDecoder` right?
If so, we also need to update its forward method to handle `position_ids`.